charts/mgmt-agent/values.yaml

```diff
@@ -60,6 +70,16 @@ deployment:
       # Files created in the Container will use group ID 2000, replace it with a different value if desired
       fsGroup: 2000
 
+   # Please provide the current epoch time in seconds (date +%s) to clean up the agent installation directory from previous deployment
```

Comment:
(Eg: Executing the following command in a bash shell will provide the epoch time: "date +%s")

Reply:
done